public/theme/boltcms-2021/partials/_master.twig

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang={{ htmllang() }}>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Work+Sans:wght@100..900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="{{ asset('css/app.css', 'theme') }}">
+    <link rel="stylesheet" href="https://unpkg.com/transition-style">
+
+    {# make sure we always display a proper title: The record's title if there is one, appended with the
+           sitename. If there is no title, we append the sitename with the payoff, if there is one. #}
+    <title>
+        {%- if record|default -%}
+            {{- record|title ~ ' | ' -}}
+        {%- endif -%}
+        {{- config.get('general/sitename') -}}
+        {%- if record|default == null and config.has('general/payoff') -%}
+            {{- ' | ' ~ config.get('general/payoff') -}}
+        {%- endif -%}
+    </title>
+    {% include 'partials/_og_tags.twig' %}
+</head>
+<body>
+{% set baseurl = app.request.getSchemeAndHttpHost %}
+{% set link = canonical()|replace({ (baseurl) : ''}) %}
+{% setcontent blockquote = 'blockquote' where { url: link } returnsingle %}
+
+{% block body %}
+    {{ include('partials/_header.twig') }}
+
+    {{ widgets('main_top') }}
+
+    {% block main %}
+        {% block main_container %}
```

Comment:
Let's keep `main` and replace references of `main_container` with `main` and BURN MAIN CONTAINER

Reply:
Done